packages/manager/modules/pci/src/projects/project/sidebar/sidebar.controller.js

```diff
@@ -37,6 +45,12 @@ export default class SidebarController {
     this.UNIVERSE = UNIVERSE;
   }
 
+  isAvailableToUser(subItem) {
```

Comment:
I wonder if it makes sense to use the same mechanism as for `regions `

Reply:
The menu is being filtered based on static `regions` (regions don't change when we switch between projects). But while filtering the menu based on users, it has to be done every time the project is switched. Hence, it cannot be done exactly the same way as the `regions` filtering. Secondly, the call to `isAvailableToUser` is a one-time binding and is going to get called only initially or when a project switch happens. So, technically, I do not see an advantage in filtering the menu the way we do it for `regions`
Please let me know you thoughts :)